.github/workflows/ci.yml

```diff
@@ -155,11 +155,12 @@ jobs:
         with:
           dry-run: true
           check-repo: false
+          ignore-unpublished-changes: true
 
   verifications-complete:
     needs:
       - cargo-verifications
-      - publish-crates-check
```

Comment:
Should this be a comment or deleted?

Reply:
ah good catch, adding it back. 

I originally set out to disable this job, but then realized it might work with `ignore-unpublished-changes: true` set.